data/src/main/scala/org/apache/predictionio/data/store/LEventStore.scala

```diff
@@ -89,6 +103,57 @@ object LEventStore {
       timeout)
```

Comment:
I think it is preferable to call the `findByEntityAsync` method from this method.

Reply:
Fixed.